Source/VMware.vSphereDSC/DSCResources/VMHostNetwork/VMHostVss/VMHostVssBridge.ps1

```diff
@@ -0,0 +1,197 @@
+    [LinkDiscoveryProtocolProtocol] $LinkDiscoveryProtocolProtocol
+
+    [void] Set() {
+        Write-Verbose -Message "$(Get-Date) $($s = Get-PSCallStack; "Entering {0}" -f $s[0].FunctionName)"
+
+        $this.ConnectVIServer()
+        $vmHost = $this.GetVMHost()
+        $this.GetNetworkSystem($vmHost)
+
+        $this.UpdateVssBridge($vmHost)
+    }
+
+    [bool] Test() {
+        Write-Verbose -Message "$(Get-Date) $($s = Get-PSCallStack; "Entering {0}" -f $s[0].FunctionName)"
+
+        $this.ConnectVIServer()
+        $vmHost = $this.GetVMHost()
+        $this.GetNetworkSystem($vmHost)
+        $vss = $this.GetVss()
+
+        if ($this.Ensure -eq [Ensure]::Present) {
+            return ($null -ne $vss -and $this.Equals($vss))
+        }
+        else {
+            $this.NicDevice = @()
+            $this.BeaconInterval = 0
+            $this.LinkDiscoveryProtocolProtocol = [LinkDiscoveryProtocolProtocol]::Unset
+
+            return ($null -eq $vss -or $this.Equals($vss))
+        }
+    }
+
+    [VMHostVssBridge] Get() {
+        Write-Verbose -Message "$(Get-Date) $($s = Get-PSCallStack; "Entering {0}" -f $s[0].FunctionName)"
+
+        $result = [VMHostVssBridge]::new()
+        $result.Server = $this.Server
+
+        $this.ConnectVIServer()
+        $vmHost = $this.GetVMHost()
+        $this.GetNetworkSystem($vmHost)
+
+        $result.Name = $vmHost.Name
+        $this.PopulateResult($vmHost, $result)
+
+        $result.Ensure = if ([string]::Empty -ne $result.VssName) { 'Present' } else { 'Absent' }
+
+        return $result
+    }
+
+    <#
+    .DESCRIPTION
+
+    Returns a boolean value indicating if the VMHostVssBridge should to be updated.
+    #>
+    [bool] Equals($vss) {
+        Write-Verbose -Message "$(Get-Date) $($s = Get-PSCallStack; "Entering {0}" -f $s[0].FunctionName)"
+
+        $typeName = 'VMware.Vim.HostVirtualSwitchBondBridge'
+        try {
+            New-Object -TypeName $typeName
+        }
+        catch {
+            Import-Module -Name 'VMware.Vim'
+        }
+        $vssBridgeTest = @()
+        if ($null -eq $vss.Spec.Bridge) {
+            $vssBridgeTest += $false
+        }
+        else {
+            $correctType = $vss.Spec.Bridge -is $typeName
+            $vssBridgeTest += $correctType
+            if ($correctType) {
+                $comparingResult = Compare-Object -ReferenceObject $vss.Spec.Bridge.NicDevice -DifferenceObject $this.NicDevice
+                $areEqual = $null -eq $comparingResult
+                $vssBridgeTest += $areEqual
+                $vssBrdigeTest += ($vss.Spec.Bridge.Beacon.Interval -eq $this.BeaconInterval)
+                $vssBridgeTest += ($vss.Spec.Bridge.LinkDiscoveryProtocolConfig.Operation -eq $this.LinkDiscoveryProtocolOperation)
+                $vssBridgeTest += ($vss.Spec.Bridge.LinkDiscoveryProtocolConfig.Protocol -eq $this.LinkDiscoveryProtocolProtocol)
+            }
+        }
+        return ($vssBridgeTest -notcontains $false)
```

Comment:
Change to NotContains for consistency.

Reply:
Is this new?
In all my previous resources I also had '-notcontains' instead of '-NotContains'

Updated, but then a number of other files will need to be updated as well.

Reply:
Yes, it seems that I forgot to add it to the StyleGuidelines file.